adafruit_fruitjam/peripherals.py

```diff
@@ -154,19 +163,24 @@ def __init__(self, audio_output="headphone", safe_volume_limit=12):
             self._buttons.append(switch)
 
         i2c = board.I2C()
-        self._dac = adafruit_tlv320.TLV320DAC3100(i2c)
+        while not i2c.try_lock():
```

Comment:
It looks like there is no `i2c.unlock()` that pairs with this `try_lock()`. That seems to be the cause of this getting stalled during initialization in the boot_animation and other code. 

I'm pretty sure we need to have an `i2c.unlock()` after this. I tested adding that line and it allows the code to continue.